components/com_contact/tmpl/category/default_items.php

```diff
@@ -189,7 +189,7 @@ class="inputbox" onchange="document.adminForm.submit();"
 		<?php endif; ?>
 
 		<?php if ($this->params->get('show_pagination', 2)) : ?>
-			<div class="com-contact-category__counter w-100">
+			<div class="com-contact-category__pagination w-100">
 				<?php if ($this->params->def('show_pagination_results', 1)) : ?>
 					<p class="com-contact-category__counter counter float-end pt-3 pe-2">
```

Comment:
```suggestion
					<p class="com-contact-category__pagination counter float-end pt-3 pe-2">
```
If you wanna be consistent, shouldn't it be changed here, too?

Reply:
Nope. If you check the 'sort-of' logic of paginations/navigations in other PHP files, you'll find this pattern:
```
<div class="xxx__navigation | xxx__pagination>
  <p class="xxxx__counter counter">...

```
Some examples:
- com_content\category\default_articles.php
- com_content\tmpl\archive\default_articles.php 
- com_content\tmpl\category\blog.php 

INMHO the extra **xxxx__counter** class of P is unnecessary anyway, it would be enough to have only "counter". In many other files the P tag has only "counter" class, 'cos the P tag is inside of the DIV tag which allows to target the P tag by any kind of CSS selector.

And if you'd change that here, you should change it in a lot other files too - just for the sake of consistency :) Which leads us to the problem of 'class name's (in)consistency' in J! as whole.

Reply:
@DelPoint You are right.